backend/src/graphql/resolver/UserResolver.ts

```diff
@@ -584,8 +550,7 @@ export class UserResolver {
 
     if (language) {
       if (!isLanguage(language)) {
-        logger.error(`"${language}" isn't a valid language`)
-        throw new Error(`"${language}" isn't a valid language`)
+        throw new LogError(`"${language}" isn't a valid language`, language)
```

Comment:
```suggestion
        throw new LogError('No  valid language', language)
```

Reply:
Since the language comes from the client it is fine to expose this information?!

Reply:
I chose `throw new LogError('Given language is not a valid language', language)`